scripts/getDependencies.pl

```diff
@@ -101,9 +101,9 @@
 		sha1 => 'bfcb96281ea3b59d626704f74bc6d625ff51cbce'
 	},
 	asmtools => {
-		url => 'https://ci.adoptopenjdk.net/view/Dependencies/job/dependency_pipeline/lastSuccessfulBuild/artifact/asmtools/asmtools.jar',
+		url => 'https://ci.adoptopenjdk.net/view/Dependencies/job/dependency_pipeline/lastSuccessfulBuild/artifact/asmtools/asmtools-core-7.0.b10-ea.jar',
 		fname => 'asmtools.jar',
-		shaurl => 'https://ci.adoptopenjdk.net/view/Dependencies/job/dependency_pipeline/lastSuccessfulBuild/artifact/asmtools/asmtools.jar.sha256sum.txt',
+		shaurl => 'https://ci.adoptopenjdk.net/view/Dependencies/job/dependency_pipeline/lastSuccessfulBuild/artifact/asmtools/asmtools-core-7.0.b10-ea.jar.sha256sum.txt',
```

Comment:
What is the difference between v7 and v8? We did not use 7 before, so should we use `asmtools-8.0.b02-ea.jar`? Also, should we run functional tests to verify the change?

Reply:
We are using 7 before https://github.com/adoptium/ci-jenkins-pipelines/pull/519. 7 is built by master branch. 8 is built by at8 branch, which I think it's a new branch? 8 is moving to build asmtools by maven. 7 is built by ant. If moving to 8 definite there will be more testing needed. That's why I'd like to keep using 7 as before for this PR.

Reply:
Talked with @sophia-guo and she confirmed that asmtools-core-7.0.b10-ea.jar is the same as asmtools.jar that we are using.